### ISSUE TITLE
possible to set folder in config.reek ?

### ISSUE THREAD

**Issue by tansaku** [troessner/reek#408]
Hey, reek is totally awesome.

I'm just getting set up with a config.reek file like so:

```yml
IrresponsibleModule:
  enabled: false
  exclude: []
```

I was wondering if there is a way to set the default directory/folder in that file, e.g. I just want reek to check lib by default, so I'd like something like:

```yml
CheckFolder: lib
IrresponsibleModule:
  enabled: false
  exclude: []
```

And than I run 'reek' instead of 'reek lib'. Is that just silly?  Wrong place perhaps?

Many thanks in advance

**Comment by troessner** [comment]
Good idea!
Actually it should be pretty easy to implement this using https://github.com/troessner/reek/blob/master/lib/reek/configuration/app_configuration.rb
Then only thing you'd then have to handle is what should happen when a path is set in the configuation but also passed via CLI.

Care for a pull request?..:)

**Comment by troessner** [comment]
We merged #550 -> closing this one.
I realize that "excluding" is not the same "including" but AFAIS this is close enough.
I'd still be open to PRs that add an explicit "include" option of course.
